EIPS/eip-1.mediawiki

```diff
@@ -13,46 +13,43 @@ EIP stands for Ethereum Improvement Proposal. A EIP is a design document providi
+The EIP process begins with a new idea for Ethereum. It is highly recommended that a single EIP contain a single key proposal or new idea. Small enhancements or patches often don't need an EIP and can be injected into the Ethereum development workflow with a patch submission to the corresponding Ethereum issue tracker. The more focused the EIP, the more successful it tends to be. The EIP editor reserves the right to reject EIP proposals if they appear too unfocused or too broad. If in doubt, split your EIP into several well-focused ones.
 
-Each EIP must have a champion -- someone who writes the EIP using the style and format described below, shepherds the discussions in the appropriate forums, and attempts to build community consensus around the idea. The EIP champion (a.k.a. Author) should first attempt to ascertain whether the idea is EIP-able. Posting to the  the [https://forum.ethereum.org/categories/protocol-and-client-discussion Protocol Discussion] forum or opening an [https://github.com/ethereum/EIP/issues Issue] is the best way to go about this.
+Each EIP must have a champion -- someone who writes the EIP using the style and format described below, shepherds the discussions in the appropriate forums, and attempts to build community consensus around the idea.
 
-Vetting an idea publicly before going as far as writing a EIP is meant to save the potential author time. Asking the Ethereum community first if an idea is original helps prevent too much time being spent on something that is guaranteed to be rejected based on prior discussions (searching the Internet does not always do the trick). It also helps to make sure the idea is applicable to the entire community and not just the author. Just because an idea sounds good to the author does not mean it will work for most people in most areas where Ethereum is used.
+Vetting an idea publicly before going as far as writing a EIP is meant to save the potential author time. Asking the Ethereum community first if an idea is original helps prevent too much time being spent on something that is guaranteed to be rejected based on prior discussions (searching the Internet does not always do the trick). It also helps to make sure the idea is applicable to the entire community and not just the author. Just because an idea sounds good to the author does not mean it will work for most people in most areas where Ethereum is used. Examples of appropriate public forums to gauge interest around your EIP include [https://www.reddit.com/r/ethereum/ the Ethereum subreddit] and [https://gitter.im/ethereum/ one of the Ethereum Gitter chat rooms].
```

Comment:
I believe that having a separate "preEIP" repo complicates matters. Reasoning: it will be difficult for editors to maintain 2 repos and keep up when a PR is ready to go. I anticipate EIP usage increasing over time, and with that there will be more to read and go over, so keeping it to one repo simplifies the processes.
However, I do think that draft EIPs should be organized somehow.  Although I like Nick's idea of requiring the cloning of the repo before submitting a PR (and this should still happen for formal submissions after initial discussions), Greg's idea of using EIP issues section as a sort of scratch pad for ideas before formerly writing the EIP is good. So here is my proposed process that combines Nick and Greg's comments.

1. Add an issue to EIP issues section to pitch idea, collect thoughts, and potentially write draft of EIP.
2. Once your EIP has had initial discussions within the issue, close the issue, clone the EIP repo, and add your EIP as a PR.
3. PR phase further refines the idea, adds implementations/examples as neccessary, and is accepted per the rules defined in EIP 1.

Moderation by the editors happens at the PR stage primarily: at that point we can view the issue for context of the EIP  and the cloned repo to add comments or corrections if needed. We can also comment on the PR itself if it is just a minor change that needs to happen.

Simplified process for EIP writers:

1. Open issue for discussion ->
2. Clone EIP repo and write formal EIP ->
3. Add the EIP as a PR to the main Ethereum repo ->
4. After discussion/implementation/etc EIP is accepted or rejected and added to the list on the README